staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1255,7 +1255,7 @@ func (c *cacheWatcher) nextBookmarkTime(now time.Time, bookmarkFrequency time.Du
 	return heartbeatTime, true
 }
 
-func getEventObject(object runtime.Object) runtime.Object {
+func getMutableObject(object runtime.Object) runtime.Object {
 	if _, ok := object.(runtime.CacheableObject); ok {
 		// It is safe to return without deep-copy, because the underlying
 		// object was already deep-copied during construction.
```

Comment:
this is no longer accurate, need to adjust this

Reply:
done